release/release_tests.yaml

```diff
@@ -4871,6 +4661,7 @@
     timeout: 28800 # 8h
     prepare: bash prepare.sh
     script: python run_gcs_ft_on_k8s.py
+    type: sdk_command
```

Comment:
why?

Reply:
old default, this test has not been migrated